### PR TITLE
feat(tracer): adopted Utility class & updated unit tests

### PR DESCRIPTION
**Pull request by dreamorosi**
## Description of your changes

This PR introduces changes to the `Tracer` module to adopt the forthcoming `Utility` class that will be released in the `@aws-lambda-powertools/commons` package.

- [x] Install released `@aws-lambda-powertools/commons` package
- [x] Make changes to the code to use the `Utility` class
- [x] Run unit & e2e tests
- [x] Add note in the docs about initialising utility outside of function handler ([ref](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/25fd4a0c2f40ff5af5ee6ec08a40a08f3e2dc22d/packages/commons/src/Utility.ts#L26))

### How to verify this change

See the checks in the PR, see the screenshot below that shows a successful e2e test run (locally):
Screenshot 2022-02-17 at 16 20 38

Optionally re-run the e2e tests on GitHub Actions.

### Related issues, RFCs

[#484](https://github.com/awslabs/aws-lambda-powertools-typescript/issues/484)  
[#547](https://github.com/awslabs/aws-lambda-powertools-typescript/pull/547)

### PR status

***Is this ready for review?:*** YES  
***Is it a breaking change?:*** NO

## Checklist

- [x] [My changes meet the tenets criteria](https://awslabs.github.io/aws-lambda-powertools-python/#tenets)
- [x] I have performed a *self-review* of my own code
- [x] My changes generate *no new warnings*
- [x] The *code coverage* hasn't decreased
- [x] I have *added tests* that prove my change is effective and works
- [x] New and existing *unit tests pass* locally and in Github Actions
- [x] Any *dependent changes have been merged and published* in downstream module
- [x] The PR title follows the [conventional commit semantics](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/main/.github/semantic.yml#L2)

### Breaking change checklist

N/A

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by dreamorosi** [comment]
This PR is on hold until #547 is merged & released.

**Review by flochaz — APPROVED** [review]

**Pull request merged**
